Manito/Manito/Screens/CreateRoom/UIComponent/CharacterCollectionView.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  ChooseCharacterView.swift
+//  Manito
+//
+//  Created by 이성호 on 2023/05/10.
+//
+
+import UIKit
+
+import SnapKit
+
+final class CharacterCollectionView: UIView {
+    
+    private enum InternalSize {
+        static let collectionHorizontalSpacing: CGFloat = 29.0
+        static let cellInterSpacing: CGFloat = 39.0
+        static let cellLineSpacing: CGFloat = 24.0
+        static let cellWidth: CGFloat = (UIScreen.main.bounds.size.width - (collectionHorizontalSpacing * 2 + cellInterSpacing * 2)) / 3
+        static let collectionInset = UIEdgeInsets(top: 0,
+                                                  left: 0,
+                                                  bottom: 0,
+                                                  right: 0)
```

Comment:
p2;

```suggestion
        static let collectionInset = UIEdgeInsets.zero
```

깔끔하게 `.zero`는 어떨까요??

Reply:
좋습니다!

[6f05570](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/455/commits/6f05570975d402b261f38fcbed3e700fd7b3357e)